index.d.ts

```diff
@@ -1,11 +1,29 @@
+import {LiteralUnion} from 'type-fest';
+
 declare namespace locatePath {
+	type OptionsType = LiteralUnion<'file' | 'directory', string>;
+
 	interface Options {
 		/**
 		Current working directory.
 
 		@default process.cwd()
 		*/
 		readonly cwd?: string;
+
+		/**
+		Type of file to match.
+
+		@default undefined
```

Comment:
I hate to do another breaking change, but I think it's worth it for clarity, to default to `'file'`.